skimage/graph/tests/test_heap.py

```diff
@@ -17,7 +17,7 @@ def _test_heap(n, fast_update):
     a = [random.uniform(1.0, 100.0) for i in range(n // 2)]
     a = a + a
 
-    t0 = time.clock()
+    t0 = time.perf_counter()
```

Comment:
What's changed here?

Reply:
apparently `time.clock` is just deprecated.

```
$ python --version                                                                                                                                                                                          
Python 3.7.1
mark2@xps ~                                                                                                                                                                                                 
$ python -c 'import time; time.clock()'                                                                                                                                                                     
-c:1: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
```